tests/em/tdem/test_TDEM_forward_Analytic.py

```diff
@@ -93,7 +93,7 @@ def halfSpaceProblemAnaDiff(
 class TDEM_SimpleSrcTests(unittest.TestCase):
     def test_source(self):
         waveform = EM.TDEM.Src.StepOffWaveform()
-        assert waveform.eval(0.) == 0.
+        assert waveform.eval(0.) == 1.
```

Comment:
I think that confusion occurs we usually think solving b. However, if you think about solving e, this  is quite clear.